addons/metrics-server/v1.8.x.yaml

```diff
@@ -136,7 +136,7 @@ spec:
         emptyDir: {}
       containers:
       - name: metrics-server
-        image: gcr.io/google_containers/metrics-server-amd64:v0.3.1
+        image: k8s.gcr.io/metrics-server-amd64:v0.3.0
```

Comment:
@zouyee any reason we're downgrading from v0.3.1 to v0.3.0 here?

Reply:
https://github.com/kubernetes-incubator/metrics-server/pull/148
https://github.com/kubernetes-incubator/metrics-server/pull/138

Reply:
From what I can tell, that bug was fixed in 0.3.1 and the image manifest simply wasn't updated in the release, but is still the latest and suggested version. I'm not clear on why we want to roll back? What am I missing 😄

Reply:
oh, my mistake